indra/newview/llselectmgr.cpp

```diff
@@ -1954,15 +1937,8 @@ void LLSelectMgr::selectionSetGLTFMaterial(const LLUUID& mat_id)
         }
     };
 
-    if (item && !item->getPermissions().allowOperationBy(PERM_COPY, gAgent.getID()))
-    {
-        getSelection()->applyNoCopyPbrMaterialToTEs(item);
```

Comment:
applyNoCopyPbrMaterialToTEs should no longer be in use, so whole function can be removed.

Also dropTextureOneFace handles protextions for textures, so similar operation can be done for applyNoCopyTextureToTEs

Reply:
I reverted the cleanup that removed this code. There were various cases that I did not account for which could cause issues with permissions.